File: C/PRAK404-2210817110006-MohammadZakiFirmansah.c
#include <stdio.h>
int main(){
    float prog, bil1, bil2;
    while (1){
    printf("\tPilih program\n");
    printf("\t1. Penjumlahan\n");
    printf("\t2. Pengurangan\n");
    printf("\t3. Perkalian\n");
    printf("\t4. Pembagian\n");
    printf("\t5. Exit\n");
    printf("Masukkan Pilihan: ");
    scanf("%f", &prog);
    if(prog > 5){
        printf("Input anda salah, silahkan coba lagi\n");
        continue;
    }else if(prog == 5){
        printf("Terimakasih, telah menggunakan kalkulator Mohammad Zaki Firmansah\n");
        break;
    }
    printf("Masukkan Nilai Pertama: ");
    scanf("%f", &bil1);
    printf("Masukkan nilai kedua: ");
    scanf("%f", &bil2);
    if(prog == 1){
        printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", bil1, bil2, bil1+bil2);
    }else if(prog == 2){
        printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", bil1, bil2, bil1-bil2);
    }else if(prog == 3){
        printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n", bil1, bil2, bil1*bil2);
    }else if(prog == 4){
        printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n", bil1, bil2, bil1/bil2);
    }
    }
}

File: Python/PRAK404-2210817110006-MohammadZakiFirmansah.py
while True :
    print("Pilih program\n 1. Penjumlahan\n 2. Pengurangan\n 3. Perkalian\n 4. Pembagian\n 5. Exit")
    prog = int(input("Masukkan Pilihan: "))
    if prog == 5 :
        print("Terimakasih, telah menggunakan kalkulator Mohammad Zaki Firmansah")
        break
    elif prog > 5 :
        print("Input anda salah, silahkan coba lagi")
        continue
    else :
        n1 =float(input("Masukkan nilai pertama: "))
        n2 =float(input("Masukkan nilai kedua: "))
        if prog == 1 :
            print("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f"% (n1, n2, n1+n2))
        elif prog == 2 :
            print("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f"% (n1, n2, n1-n2))
        elif prog == 3 :
            print("Hasil perkalian antara %.2f dengan %.2f adalah %.2f"% (n1, n2, n1*n2))
        elif prog == 4 :
            print("Hasil pembagian antara %.2f dengan %.2f adalah %.2f"% (n1, n2, n1/n2))